plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/product-details/style.scss

```diff
@@ -1,38 +1,72 @@
-.wp-block-woocommerce-product-details {
-	ul.wc-tabs {
-		list-style: none;
-		padding: 0 0 0 1em;
-		margin: 0 0 1.618em;
-		overflow: hidden;
-		position: relative;
-		border-bottom: 1px solid $gray-200;
+/*
+ * Minimal style.
+ */
+body .wp-block-woocommerce-product-details.is-style-minimal {
```

Comment:
Do we need `body` here because we want to raise the specificity of this selector? If so, I think it's a good idea to mention that in a comment, to make sure somebody refactoring this file in the future won't remove it by mistake.

Reply:
> Do we need a body here because we want to raise the specificity of this selector?

Yes.

The selector here is tricky. We need it to work in both the editor _and_ the front-end context, and to be specific enough to override core's rules being at specificity (0,4,3) `.woocommerce div.product .woocommerce-tabs ul.tabs li`. Having said that, looking at the hover-related issue you mentioned https://github.com/woocommerce/woocommerce/pull/42965#discussion_r144585600 I think my last refactor needs to be corrected.

The current CSS selector specificity (0,4,3) `body .wp-block-woocommerce-product-details.is-style-minimal ul.tabs.wc-tabs` only works because blocks enqueue the CSS file _later_ than core does.

I fixed it in https://github.com/woocommerce/woocommerce/pull/42965/commits/de036db07a7ca544636442bcfee3099763e96bea by introducing a (0,4,4) `html body .wp-block-woocommerce-product-details.is-style-minimal ul.tabs.wc-tabs` specificity, although it is difficult to be 100% error-proof when it comes to existing theme-based overrides. We probably need to keep an eye out for conflicts!

I've also added an explanation on the doc block. Thanks for the suggestion!